torch/_export/serde/serialize.py

```diff
@@ -145,9 +195,34 @@ def serialize_operator(target) -> str:
         return str(target)
 
 
+def deserialize_operator(serialized_target: str):
+    target = torch.ops
+    for name in serialized_target.split("."):
+        if not hasattr(target, name):
+            log.warning(f"Could not find operator {serialized_target}. Returning fake operator.")  # noqa: G004
+
+            # Create a random fake placeholder op
+            def fake_op(x):
+                return x
+            fake_op.__name__ = serialized_target
+            return fake_op
```

Comment:
oh shouldn't we just error out?

Reply:
For SDK use case, they don't care if the operator exists, they just need a string name.